rgw/standalone/s3cmd-setup-v3-auto-dfg.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
+'''
+> download this script for auto configure s3cmd and create a bucket and upload 3 types of files.
+> use - chmod +x script_name.py and then run - ./script_name.py
+> compatible with python version 3
+> script functionalities
+        - enable epel
+        - install python2-pip, upgarde pip, install s3cmd
+        - create a user = 'operator'
+        - configure .s3cfg file
+        - create 3 files(500kb, 500mb, 4gb) and upload that in bucket (with 50000 objects)
+        - delete 3 files created in the system
+'''
+import os, sys, random
+print('enable the epel repo or by default it may be installed')
+enable_epel = os.system('yum-config-manager --enable epel')
+print(enable_epel)
+print("----------------------------------------------------------------------")
+install_python3_pip = os.system('yum install python3-pip -y')
+print(install_python3_pip)
+print("----------------------------------------------------------------------")
```

Comment:
since this statement is used many times, i would suggest to create a definition for it and use character replication statements, something like,

```
def marker(char="-", length=30):
   print(char*length)
   # or with logger
   log.info(char*length)
```
 
```
$ python -c "print('*'*30)"                                                                                                                                                               ─╯
******************************
```